tests/cli/net_handler_cli.c

```diff
@@ -345,21 +360,51 @@ void connect_keyboard() {
     }
 }
 
-int main() {
+int main(int argc, char** argv) {
     char nextcmd[MAX_CMD_LEN];  // to hold nextline
     int stop = 0;
 
     signal(SIGINT, sigint_handler);
 
+    // Parse command line flags
+    int c;
+    opterr = 0;
+    while ((c = getopt(argc, argv, "ds")) != -1) {
+        switch (c) {
+            case 'd':
+                spawned_net_handler = 0;
+                dawn = 1;
```

Comment:
Should these variables be of the type `bool` that you defined in `runtime_util`? I didn't realize you could do that, we should change all of our flag variables to this type, kinda like a "pseudo-boolean" type. We should make sure that the `bool` type  you defined takes on the underlying type of `uint8_t` though. Not sure how to do that.